### PR TITLE
Add -u flag, for unbuffered output.

### PR DESCRIPTION
**Pull request by jonsimantov**
I prefer this change to the other PR.

### CODE REVIEW

File: demumble.cc [hunk 1]
```diff
@@ -113,8 +116,9 @@ int main(int argc, char* argv[]) {
         size_t special = strcspn(cur, "_?");
         if (print_mode == kPrintAll)
           printf("%.*s", static_cast<int>(special), cur);
-        else if (need_separator)
+        else if (need_separator) {
```
Comment (nico) [inline demumble.cc]: please revert this change

Comment (jonsimantov) [reply demumble.cc]: Is this extra commit OK or would you like a fresh branch with this squashed?

File: demumble.cc [hunk 2]
```diff
@@ -115,6 +118,7 @@ int main(int argc, char* argv[]) {
           printf("%.*s", static_cast<int>(special), cur);
         else if (need_separator)
           printf("\n");
+
```
Comment (nico) [inline demumble.cc]: Can you just not touch the code down here at all please? :-)

**Comment by nico** [comment]
(and squash commits into a single commit too please)

File: demumble.cc (hunk 2 above)
Comment (jonsimantov) [reply demumble.cc]: Heh, sorry. I'll send a fresh PR.
